newIDE/app/src/AssetStore/AssetDetails.js

```diff
@@ -268,6 +312,22 @@ export const AssetDetails = React.forwardRef<Props, AssetDetailsInterface>(
                           </Link>
                         );
                       })}
+                    {!!authorPublicProfiles.length &&
```

Comment:
Should it be a ternary with above component to avoid having multiple links?

Reply:
Well I thought this could allow having assets with a mix of external links and GDevelop users (even if it's not the case today)

Reply:
I'm not sure the display would be great (no spacer I think)

Reply:
I've just fixed this, cf screenshot